homeassistant/components/sensor/zha.py

```diff
@@ -143,3 +146,20 @@ def state(self):
             return None
 
         return round(float(self._state))
+
+
+class IlluminanceMeasurementSensor(Sensor):
+    """ZHA lux sensor."""
+
```

Comment:
No value attribute here?

Reply:
it's defined as 0 in parent class Sensor